crates/pop-cli/src/commands/build/parachain.rs

```diff
@@ -19,8 +19,10 @@ pub struct BuildParachainCommand {
 impl BuildParachainCommand {
 	pub(crate) fn execute(&self) -> anyhow::Result<()> {
 		clear_screen()?;
-		intro(format!("{}: Building a parachain", style(" Pop CLI ").black().on_magenta()))?;
+		intro(format!("{}: Building your parachain", style(" Pop CLI ").black().on_magenta()))?;
 		set_theme(Theme);
+
+		warning("NOTE: this may take some time...")?;
```

Comment:
I like this. Just a comment, ideally I'd love to see the same as you implemented for `pop up`: https://github.com/r0gue-io/pop-cli/pull/123